deploy/helm/flightctl/charts/ui/templates/flightctl-ui-cm.yaml

```diff
@@ -7,6 +7,9 @@ data:
   FLIGHTCTL_SERVER: {{ .Values.api.url | quote }}
   FLIGHTCTL_GRPC_SERVER: {{ .Values.api.grpcUrl | quote }}
```

Comment:
Could you please remove `FLIGHTCTL_GRPC_SERVER` ?

Reply:
great catch. Updated